src/table/internal.tsx

```diff
@@ -156,6 +157,19 @@ const InternalTable = React.forwardRef(
     const hasSelection = !!selectionType;
     const hasFooter = !!footer;
 
+    const noStickyColumns = !stickyColumns?.first && !stickyColumns?.last;
```

Comment:
minor: negated boolean variables take extra time to "process" in our heads. can we make this `hasStickyColumns` like the other variables above?